src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```

lets do not use static methods here https://medium.com/att-israel/should-you-avoid-using-static-ae4b58ca1de5